lite-rpc/src/bridge.rs

```diff
@@ -563,11 +563,7 @@ impl LiteRpcServer for LiteBridge {
                         ui_accounts.push(ui_account);
                     }
                     Err(_) => {
-                        // internal error while fetching multiple accounts
-                        return Err(jsonrpsee::types::error::ErrorCode::ServerError(
-                            RpcErrors::AccountNotFound as i32,
-                        )
-                        .into());
+                        ui_accounts.push(None);
```

Comment:
should assert ui_accounts.len()==pubkey_strs.len();
we should do that for all vector/batched request-response pairs!